jabberbot.py

```diff
@@ -107,10 +84,15 @@ def muc_message(self, msg):
                    for stanza objects and the Message stanza to see
                    how it may be used.
         """
+
+        # "parse" body and choose correct callback
+        # Just debugging! This needs to be done properly!
         if msg['mucnick'] != self.nick and self.nick in msg['body']:
-            self.send_message(mto=msg['from'].bare,
-                              mbody="I heard that, %s. But I'm stupid now, waiting for your Pullrequest at https://github.com/ombre8/jabberbot" % msg['mucnick'],
-                              mtype='groupchat')
+            onSysbotMentioned(msg)
+        if msg['mucnick'] != self.nick and "meme" in msg['body']:
+            onMemeRequested(msg)
+        if msg['mucnick'] != self.nick and "Tagesverantworung" in msg['body']:
```

Comment:
Why is there already a TV function? And why do I have to type "Tagesverantworung" and not "Tagesverantwor**t**ung" or just TV